perceval/backend.py

```diff
@@ -253,6 +267,10 @@ def metadata_updated_on(item):
     def metadata_category(item):
         raise NotImplementedError
 
+    @staticmethod
+    def search_terms(item):
```

Comment:
Wouldn't it be better to use the same behaviour we have for `classified_fields`?.

We have a common method in `Backend` class that removes those fields. They are defined in `CLASSIFIED_FIELDS` class variable. I think it will be better to have something similar, so we just need a piece of code to copy these fields and then each backend would define which fields are going to be included.

Reply:
I will give it a try